kotlin-android/app/src/main/java/io/realm/examples/objectserver/model/Task.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.examples.objectserver.model
+
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import org.bson.types.ObjectId
+
+enum class TaskStatus(val text: String) {
+    OPEN("Open"),
+    IN_PROGRESS("In Progress"),
+    COMPLETE("Complete")
+}
+
+open class Task(user_id: String = "default", _name: String = "default") : RealmObject() {
```

Comment:
No, this should be projectId and the default should be `"My Project"` (so we can see the two apps sync)

Reply:
I'm not entirely sure what you mean here. I'll message you about this.